cmd/lncli/walletrpc_active.go

```diff
@@ -848,6 +848,14 @@ var fundTemplatePsbtCommand = cli.Command{
 				"if required",
 			Value: -1,
 		},
+		cli.StringFlag{
+			Name: "coin_selection_strategy",
+			Usage: "(optional) the strategy to use for selecting " +
+				"coins to fund the PSBT from a template. " +
+				"If provided it will override the globally " +
+				"configured strategy in lnd.conf",
```

Comment:
should mention which options are available. 

Also, since this same flag is used in many places, might be worth extracting into a re-usable variable?